### DISCUSSION TITLE
Atomic Bitwise Operation

### DISCUSSION THREAD

**Discussion by nguyen4dbkvn** [prisma/prisma#5760] [Q&A] [answered]
### Problem
I need to query like:
```sql
SELECT * 
FROM resourcepermission 
WHERE
	actionIds & (SELECT bitwise FROM resourceaction where NAME = ? AND actionId = ?)
	AND EXISTS (
		SELECT roleId FROM users_roles 
			WHERE users_roles.roleId = resourcepermission.roleId AND users_roles.userId = ?
	)
	AND name = ? AND primKey = ?
```
I haven't found the document to execute atomic bitwise operation.
Can I execute this query by Prisma without using raw query?
Thank you for helping.

**Comment by ryands17** [comment, marked as answer]
@nguyen4dbkvn 👋 

Unfortunately we do not support using Bitwise operators directly with Prisma so you would need to use [raw queries](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access) for the time being.

I would suggest creating a feature request for the same [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) so that we can look into this :)

**Reply by nguyen4dbkvn** [reply]
Thank you for your reply. 
I will move it to feature request.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
